community/examples/AMD/hpc-cluster-amd-slurmv5.yaml

```diff
@@ -0,0 +1,125 @@
+  region: us-east4
+  zone: us-east4-c
+
+deployment_groups:
+- group: primary
+  modules:
+  - source: modules/network/vpc
+    id: network1
+  - source: modules/file-system/filestore
+    id: homefs
+    use: [network1]
+    settings:
+      local_mount: /home
+  - source: modules/file-system/filestore
+    id: swfs
+    use: [network1]
+    settings:
+      local_mount: /sw
+  - source: community/modules/scripts/spack-install
+    id: spack
+    settings:
+      install_dir: /sw/spack
+      spack_url: https://github.com/spack/spack
+      spack_ref: v0.18.1
+      configs:
+      - type: file
+        scope: site
+        content: |
+          packages:
+            slurm:
+              externals:
+                - spec: slurm@22-05-2
+                  prefix: /usr/local
+              buildable: False
+      log_file: /var/log/spack.log
+      compilers: []
+      packages: []
+  - source: modules/scripts/startup-script
+    id: spack-startup
+    settings:
+      runners:
+      - type: shell
+        source: modules/startup-script/examples/install_ansible.sh
+        destination: install_ansible.sh
+      - type: ansible-local
+        source: modules/spack-install/scripts/install_spack_deps.yml
+        destination: install_spack_deps.yml
+      - type: shell
+        content: $(spack.startup_script)
+        destination: install_spack.sh
+      # the following installation of AOCC may be automated in the future
+      # with a clear direction to the user to read the EULA at
+      # https://developer.amd.com/aocc-compiler-eula/
+      - type: data
+        destination: /var/tmp/install_aocc.sh
+        content: |
+          #!/bin/bash
+          spack install aocc@3.2.0 +license-agreed
+          spack load aocc@3.2.0
+          spack compiler find --scope site
+          spack -d install -v openmpi@4.1.3 %aocc@3.2.0 +legacylaunchers +pmi schedulers=slurm
+  - source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    id: low_cost_partition
+    use:
+    - network1
+    - homefs
+    - swfs
+    settings:
+      partition_name: lowcost
+      machine_type: c2d-standard-4
+      node_count_dynamic_max: 10
+      enable_placement: false
+      is_default: false
+  - source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    id: compute_partition
+    use:
+    - network1
+    - homefs
+    - swfs
+    settings:
+      partition_name: compute
+      machine_type: c2d-standard-112
+      node_count_dynamic_max: 50
+      enable_placement: true
+      is_default: true
+  - source: community/modules/scheduler/schedmd-slurm-gcp-v5-controller
+    id: slurm_controller
+    use:
+    - network1
+    - homefs
+    - swfs
+    - low_cost_partition  # low cost partition will be default as it is listed first
+    - compute_partition
+    settings:
+      machine_type: c2d-standard-4
+  - source: community/modules/scheduler/schedmd-slurm-gcp-v5-login
+    id: slurm_login
+    use:
+    - network1
+    - homefs
+    - swfs
```

Comment:
The login node does not have a `network_storage` parameter, so `homefs` and `swfs` are not being used by this module. They can be removed as this is handled by the controller already.